src/cripts/Bundles/Common.cc

```diff
@@ -105,20 +105,18 @@ Common::via_header(const Cript::string_view &destination, const Cript::string_vi
 }
 
 Common &
-Common::set_config(const Cript::string_view name, const Cript::Records::ValueType value)
+Common::set_config(const Cript::string_view name, const Cript::Records::ValueType &value)
 {
-  Cript::Records rec(name);
+  auto rec = Cript::Records::lookup(name); // These should be loaded at startup
```

Comment:
need to check rec to make sure its valid

Reply:
That's actually done by the lookup() method. I can change this though, which seems sensible.

Reply:
Done